### ISSUE TITLE
Field validation fails for objects loaded from fixtures if required field has_rich_text

### ISSUE THREAD

**Issue by mireksawicz** [rails/rails#37815]
### Steps to reproduce
- Create a new rails app
`rails new acton_text`
- Generate a new model with a text field
`rails g model Comment comment:text`
- Add the following to the `Comments` model
```
  has_rich_text :comment
  validates_presence_of :comment
```
- Create the following test in `test/models/comment_test.rb`
```
  test 'should be valid if comment is present' do
    comment = comments(:one)
    assert 'MyText' == comment.read_attribute(:comment), 'comment text not present'
    assert comment.valid?, 'comment text present but record still not valid'
  end
```
- Install action_text
`rails action_text:install`
- create database and run migrations
`rails db:create RAILS_ENV=test`
`rails db:migrate RAILS_ENV=test`
- then run the test
`rails test test/models/comment_test.rb`

### Expected behavior
I expected the test to pass since the required field is present in the fixtures and reading the attribute shows that it is present.

### Actual behavior
The test fails and when in debut asking the comment for its comment field it returns `nil`
`comment.comment.body => nil` 

### System configuration
**Rails version**: 6.0.0 and 6.0.1

**Ruby version**: 2.5.7

This is my first ever bug report to the Rails community, so if I did something wrong please let me know and please be gentle :)

**Comment by kuei0221** [comment]
Hi @mireksawicz ,

I have reproduced your process and get the following error message: 
`comment text present but record still not valid` , 
and also the nil value as you have mentioned.
Actually, the ActionText::RichText here is a new record and created without any body value.
```
(byebug) comment.comment
#<ActionText::RichText id: nil, name: "comment", body: nil, record_type: "Comment", record_id: 980190962, created_at: nil, updated_at: nil>
(byebug) comment.comment.nil?
true
(byebug) comment.comment.new_record?
true
```
I have also tried it in the development env, and it works perfectly.
```
Comment.create(comment: "x")
begin transaction
 Comment Create (3.4ms)  INSERT INTO "comments" ("created_at", "updated_at") VALUES (?, ?)  [["created_at", "2019-11-29 10:01:53.281909"], ["updated_at", "2019-11-29 10:01:53.281909"]]
 ActionText::RichText Create (0.3ms)  INSERT INTO "action_text_rich_texts" ("name", "body", "record_type", "record_id", "created_at", "updated_at") VALUES (?, ?, ?, ?, ?, ?)  [["name", "comment"], ["body", "x"], ["record_type", "Comment"], ["record_id", 4], ["created_at", "2019-11-29 10:01:53.307840"], ["updated_at", "2019-11-29 10:01:53.307840"]]
ActiveStorage::Attachment Load (0.1ms)  SELECT "active_storage_attachments".* FROM "active_storage_attachments" WHERE "active_storage_attachments"."record_id" = ? AND "active_storage_attachments"."record_type" = ? AND "active_storage_attachments"."name" = ?  [["record_id", 4], ["record_type", "ActionText::RichText"], ["name", "embeds"]]
Comment Update (0.2ms)  UPDATE "comments" SET "updated_at" = ? WHERE "comments"."id" = ?  [["updated_at", "2019-11-29 10:01:53.310967"], ["id", 4]]
 commit transaction
 => #<Comment id: 4, comment: nil, created_at: "2019-11-29 10:01:53", updated_at: "2019-11-29 10:01:53">
```
Look closely from the Query, the "x" value only exist in ActionText::RichText, but not Comment itself.
So if the test env works the same as dev env, your first test should fail as well.

Therefore, I will assume that the fixture has a bug when creating ActionText::RichText's body attribute. It seems stop before the body is done.

First bug report as well, hopefully I have give enough information for your further works.  :)

**Comment by mireksawicz** [comment]
Hmm... I have done some more checks and I think I got the wrong end of the stick with the action text and rich_text_fields. 
I started digging into how fixtures are loaded and how the comments field is populated, but then I noticed that the `has_rich_text` field is not stored in the same table with the Comment model. just like @kuei0221 spotted. 
The `ActionText` fields are stored in a fixture `rich_texts.yml` in the following format;
```
one:
  record: one (Comment)
  name: description
  body: <p>There is text here</p>
```
I got confused about how the action text works and made an incorrect assumption. 
I'm gonna close this issue now since there is nothing to be done.
